docs/preview/features/secret-store/azure-functions.md

```diff
@@ -0,0 +1,74 @@
+---
```

Comment:
Just a thought but maybe we should just move it to the other doc as well in a dedicated header similar to `Using secret store with Azure Functions`?

Because the only things that are different here are:
1. NuGet 📦 
2. `Startup` sample
3. Consumption example

All the rest is the same or not?

Reply:
Yes, all the same; it's maybe easier to point to a dedicated page for this. IDK, I was thinking of other environments too where we have to create a dedicated extension for it.
But otherwise, yeah, I can agree on this.

Reply:
Let's start with all in one place because it's easier to split later on than merge. I want to avoid the "spread of information"

Reply:
Yes, done that.